src/plusplus.coffee

```diff
@@ -33,7 +33,9 @@ module.exports = (robot) ->
     # from beginning of line
     ^
     # the thing being upvoted, which is any number of words and spaces
-    ([\s\w'@.-]*)
+    ([\s\w'@.-:]*)
```

Comment:
Also, that will cause matches like `turn:down++ for what` to fail if not included in the matcher. Maybe that's desired?